packages/atomic/src/pages/examples/ipx.html

```diff
@@ -20,6 +28,35 @@
         searchInterface.i18n.addResourceBundle('en', 'caption-filetype', {
           '.html': 'html',
         });
+      };
+
+      (async () => {
+        await customElements.whenDefined('atomic-search-interface');
+        const searchInterface = document.querySelector('atomic-search-interface');
+
+        const ipxInSelector = searchInterface.cloneNode(true);
+
+        await initializeSearchInterface(searchInterface);
+
+        const hostDiv = document.querySelector('div#host-div');
+        hostDiv.style.display = 'none';
+        hostDiv.appendChild(ipxInSelector);
+        ipxInSelector.querySelector('atomic-ipx-modal').setAttribute('is-embedded', 'true');
+        ipxInSelector.removeChild(ipxInSelector.querySelector('atomic-ipx-button'));
+        await initializeSearchInterface(ipxInSelector);
```

Comment:
I think this (and the `cloneNode` call above) should be extracted into a separate function, like `createEmbeddedInterfaceClone`.